INSTALL.rst

```diff
@@ -221,8 +246,139 @@ Commands can be executed using:
 
     $ python src/manage.py <command>
 
-There are no specific commands for the project. See
-`Django framework commands`_ for all default commands, or type
-``python src/manage.py --help``.
+You can always get a full list of available commands by running:
+
+.. code-block:: bash
+
+    $ python src/manage.py help
+
+There are a number of developer management commands available in this project.
+
+``appointment``
+---------------
+
+Performs various appointment plugin calls.
+
+``dmn_evaluate``
+----------------
+
+Evaluate a particular decision definition.
+
+``dmn_list_definitions``
+------------------------
+
+List the available decision definitions for a given engine.
+
+``check_duplicate_component_keys``
+----------------------------------
+
+Check all forms and report duplicated component keys.
+
+``export``
+----------
```

Comment:
Just curious if we can't generate this list using sphinx stuff, and move it to docs? I doubt we are going to keep this up to date...

Reply:
I think Django has a custom sphinx extension for that, so we could look into that!